shenyu-examples/shenyu-examples-motan/shenyu-examples-motan-service/src/main/java/org/apache/shenyu/examples/motan/service/impl/MotanDemoServiceImpl.java

```diff
@@ -25,6 +25,7 @@
  * Motan demo service.
  */
 @MotanService(export = "demoMotan:8002")
+@ShenyuMotanClient(path = "/demo/")
```

Comment:
Please also consider the situation of `@ShenyuMotanClient(path = "/demo/**")`

Reply:
It is suggested to add a new test class. If you modify the existing test class, you will also need to modify `shenyu-integrated-test-motan` module.

Reply:
ok,thanks